README.md

```diff
@@ -1,8 +1,19 @@
 # Virtual MMX
 
 [![](https://img.shields.io/github/issues/wintergatan-community/virtual-mmx)](https://github.com/wintergatan-community/virtual-mmx/issues)
- [![](https://img.shields.io/github/issues-pr/wintergatan-community/virtual-mmx)](https://github.com/wintergatan-community/virtual-mmx/pulls)
- [![](https://img.shields.io/badge/contribute-info-blue?style=flat-square)](https://github.com/wintergatan-community/virtual-mmx/blob/master/CONTRIBUTING.md)
- [![Discord](https://img.shields.io/badge/Discord-join-7289DA?logo=discord&style=flat-square)](https://discord.gg/rMK6DFT)
+[![](https://img.shields.io/github/issues-pr/wintergatan-community/virtual-mmx)](https://github.com/wintergatan-community/virtual-mmx/pulls)
+[![](https://img.shields.io/badge/contribute-info-blue?style=flat-square)](https://github.com/wintergatan-community/virtual-mmx/blob/master/CONTRIBUTING.md)
+[![Discord](https://img.shields.io/badge/Discord-join-7289DA?logo=discord&style=flat-square)](https://discord.gg/rMK6DFT)
 
-A WebApp to program songs on a virtual Marble Machine X
+## About
+
+A WebApp to program songs on a virtual Marble Machine X. Songs can be exported and stored on local disk.
+
+To view the future plans we have for this WebApp, please visit our [planning board](https://github.com/orgs/wintergatan-community/projects/1).
+
+Exported data is using the structure described in the [vmmx schema](https://github.com/wintergatan-community/vmmx-schema) github repository.
+This repository also contains the definition for our internal data structure.
```

Comment:
It’s in the vmmx-schema repository. With my comment above, I intended for line 14 to be rephrased to something like „Internal data structure is described in ...“